### DISCUSSION TITLE
Integrate with existing non gpt-pilot projects.

### DISCUSSION THREAD

**Discussion by KaurKadakWise** [Pythagora-io/gpt-pilot#21] [Ideas]
Could be awesome to see functionality where gpt-pilot crawls through an already existing project, and helps with new feature development. Would add so much value. Is this something that is possible to integrate with the current architecture?

**Comment by zvone187** [comment]
Yes, definitely. The main thing that needs to be done is enable user to input the business logic behind the app. Then, GPT Pilot would analyse all files and the code and predict what each file is used for and once that's confirmed, gpt pilot would be able to resume app development just like now. Unfortunately, we're super low on resources so won't be able to test this out soon but if you would like to try it out, let me know and I can help you get familiar with the code.

**Reply by nalbion** [reply]
https://github.com/Pythagora-io/gpt-pilot/pull/59

**Reply by nalbion** [reply]
#65 - load app by user workspace

**Reply by LeonOstrez** [reply]
Thank you @nalbion ! Merged.
